docs/guides/deployment/azure_flexibleserver.rst

```diff
@@ -0,0 +1,157 @@
+.. _ref_guide_deployment_azure_flexibleserver:
+
+================================
+On Azure Postgres FelxibleServer
+================================
+
+In this guide we show how to deploy EdgeDB on Azure using Postgres
+FelxibleServer as the backend.
+
+
+Prerequisites
+=============
+
+* Valid Azure Subscription with billing enabled or credits (`free trial
+  <azure-trial_>`_).
+* Azure CLI (`install <azure-install_>`_).
+
+.. _azure-trial: https://azure.microsoft.com/en-us/free/
+.. _azure-install: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli
+
+
+Provision an EdgeDB instance
+============================
+
+Login to your azure account.
+
+.. code-block:: bash
+
+   $ az login
+
+Create a new resource group.
+
+.. code-block:: bash
+
+   $ GROUP=my-group-name
+   $ az group create --name $GROUP --location westus
+
+Provision a PostgreSQL server.
+
+.. note::
+
+   If you already have a database provisioned you can skip this step.
+
+.. note::
+
+   If you get an error saying ``Specified server name is already used.`` change
+   the server name an rerun the command.
+
+.. code-block:: bash
+
+   $ PG_SERVER_NAME=postgres-for-edgedb
+   $ PG_PASSWORD=SomeSecret1
```

Comment:
let`s use `read -s PG_PASSWORD` to avoid having passwords in plaintext in shell history